piksi_tools/console/settings.yaml

```diff
@@ -815,7 +815,7 @@
   expert: false
   type: integer
   units: us (microseconds)
-  default value: 0
+  default value: '0'
```

Comment:
Is it correct to set this as string if type is integer?

Reply:
In other case we will not see the value in Piksi console in Settins tab.
Look at simulator::speed_sigma for example. I tried to do by analogy.

Reply:
ok

Reply:
@denniszollo could you advice about this? (string vs int)